contracts/schemes/GenericSchemeMultiCall.sol

```diff
@@ -0,0 +1,167 @@
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
```

Comment:
why this is needed ?

Reply:
@orenyodfat bytes[]  _callData is only supported with ABIEncoderV2 in 0.5

Reply:
ok. the other alternative is to pass in bytes _callData which is concatenation of call data of all calls .
and provide additional lens array to parse it.
something like https://github.com/daostack/arc/blob/master-2/contracts/utils/DAOFactory.sol#L235 
this will eliminate the need to use the `experimental` pragma which is not experimental.

Reply:
@orenyodfat the DAOfactory is also using ABIEncoderV2, so I guess that shouldn't be an issue

Reply:
DAOFactory use solidity 6.12